shoes-swt/lib/shoes/swt/app.rb

```diff
@@ -154,6 +154,14 @@ def wait_until_closed
 
       private
 
+      def initialize_background
+        @background = if @dsl.opts[:background]
+          Color.new(@dsl.opts[:background]).real
+        else
+          Color.system_background_color
```

Comment:
If we're including the background in the `DEFAULT_OPTIONS` above, do we hit this branch in practice? Would be great to avoid the conditional if we could know we're going to get a color set  in the DSL.